staging_service/autodetect/GenerateMappings.py

```diff
@@ -28,223 +28,114 @@
+    media_id: {
         "title": "Media",
         "app": "kb_uploadmethods/import_tsv_excel_as_media_from_staging",
         "output_type": ["KBaseBiochem.Media"],
     },
-    # Commented out because: Not in scope and requires an object
-    # {
-    #     "title": "Attribute Mapping",
-    #     "app": "kb_uploadmethods/import_attribute_mapping_from_staging",
-    #     "output_type": ["KBaseExperiments.AttributeMapping"],
-    # },
-    {
-        "id": expression_matrix_id,
+    expression_matrix_id: {
         "title": "Expression Matrix",
         "app": "kb_uploadmethods/import_tsv_as_expression_matrix_from_staging",
         "output_type": ["KBaseFeatureValues.ExpressionMatrix"],
     },
-    {
-        "id": metabolic_annotations_id,
+    metabolic_annotations_id: {
         "title": "Metabolic Annotations",
         "app": "MergeMetabolicAnnotations/import_annotations",
         "output_type": ["KBaseGenomes.Genome"],
     },
-    {
-        "id": metabolic_annotations_bulk_id,
+    metabolic_annotations_bulk_id: {
         "title": "Bulk Metabolic Annotations",
         "app": "MergeMetabolicAnnotations/import_bulk_annotations",
         "output_type": ["KBaseGenomes.Genome"],
     },
-    {
-        "id": fba_model_id,
+    fba_model_id: {
         "title": "FBA Model",
         "app": "kb_uploadmethods/import_file_as_fba_model_from_staging",
         "output_type": ["KBaseFBA.FBAModel"],
     },
-    {
-        "id": phenotype_set_id,
+    phenotype_set_id: {
         "title": "Phenotype Set",
         "app": "kb_uploadmethods/import_tsv_as_phenotype_set_from_staging",
         "output_type": ["KBasePhenotypes.PhenotypeSet"],
     },
-]
-
-mapping[EXCEL] = [
-    {
-        "id": sample_set_id,
-        "title": "Samples",
-        "app": "sample_uploader/import_samples",
-        "output_type": ["KBaseSets.SampleSet"],
-    },
-    {
-        "id": media_id,
-        "title": "Media",
-        "app": "kb_uploadmethods/import_tsv_excel_as_media_from_staging",
-        "output_type": ["KBaseBiochem.Media"],
-    },
-    {
-        "id": attribute_mapping_id,
+    attribute_mapping_id: {
         "title": "Attribute Mapping",
         "app": "kb_uploadmethods/import_attribute_mapping_from_staging",
         "output_type": ["KBaseExperiments.AttributeMapping"],
     },
-    {
-        "id": fba_model_id,
-        "title": "FBA Model",
-        "app": "kb_uploadmethods/import_file_as_fba_model_from_staging",
-        "output_type": ["KBaseFBA.FBAModel"],
-    },
-]
-mapping[JSON] = [
-    {
-        "id": escher_map_id,
+    escher_map_id: {
         "title": "EscherMap",
         "app": "kb_uploadmethods/import_eschermap_from_staging",
         "output_type": ["KBaseFBA.EscherMap"],
-    }
-]
+    },
 
-mapping[SBML] = [
-    {
-        "id": fba_model_id,
-        "title": "FBA Model",
-        "app": "kb_uploadmethods/import_file_as_fba_model_from_staging",
-        "output_type": ["KBaseFBA.FBAModel"],
-    }
-]
+}
+
+mapping = {}
```

Comment:
maybe we can rename this to file_format_to_app_mapping?

Reply:
Done